jina/serve/runtimes/worker/http_fastapi_app.py

```diff
@@ -1,5 +1,7 @@
 import inspect
 from typing import TYPE_CHECKING, Callable, Dict, List, Optional, Union
+import gradio as gr
```

Comment:
this would imply thay impprting gradio should be protected, and the feature should be activated via an argument for Deployment or Flow

Reply:
got it, I will move it inside the function call or if block